README.md

```diff
@@ -3,46 +3,85 @@
 [![Tests](https://github.com/wmo-im/csv2bufr/workflows/tests%20%E2%9A%99%EF%B8%8F/badge.svg)](https://github.com/wmo-im/csv2bufr/actions/workflows/tests.yml)
 
 csv2bufr is a Python package to transform CSV data into WMO BUFR.  Currently, csv2bufr converts each row in a 
-CSV to a BUFR message.
+CSV to a BUFR message. Both a python module and CLI to the module are provided.
 
-### Usage
+### Installation
+The csv2bufr module relies on the ECCODES software library from ECMWF. 
+Prior to installing the module ECCODES needs to be installed. 
+As part of the repository this is done via a Docker image, this is currently built as part of the install process but future versions will rely on a pre-built image. 
+
+Clone the repository and enter the csv2bufr directory, working from the dev branch.  
 
 ```bash
-# first build the Docker image
-docker build -t eccodes_v23 .
+# clone from github
+git clone https://github.com/wmo-im/csv2bufr.git -b dev
+cd csv2bufr
+# build using docker
+docker build -t cvs2bufr .
+# run (note the volume is not required but has been included whilst in development)
+docker run -it -v ${pwd}:/app csv2bufr
+```
+
+The software should now be installed and ready to run.
 
-# now start the Docker container based on the image
-docker run -it -v ${pwd}:/app eccodes_v23
+### Configuration
 
-# export path to Python modules for this script
-export PYTHONPATH="${PYTHONPATH}:/app/src/csv2bufr/"
+Two configurations files are required, the first specifying how to map from the input CSV file to BUFR.
+The second contains the station level metadata from the WMO OSCAR/Surface catalogue. 
+Both files take the JSON format and a description of the mapping file is given below under usage.
+The second can be downloaded by the pyoscar tool (installed as part of the image build).
```

Comment:
....can be downloaded by the [pyoscar](https://github.com/wmo-cop/pyoscar) tool ....

Reply:
done